azext_iot/central/services/destination.py

```diff
@@ -154,7 +153,7 @@ def list_destinations(
         )
 
         if "value" not in result:
-            raise CLIError("Value is not present in body: {}".format(result))
+            raise CLIInternalError("Value is not present in body: {}".format(result))
```

Comment:
Should be AzureResponseError

Reply:
Gotcha, will update these - I think I may have gotten some of them confused with the `Error parsing body` issues elsewhere. In this instance, we're expecting the service response to have `value` as a property, and if it doesn't we're assuming that implies a service error.